Add Gallery by group

JuJuMDX opened the pull request:
We can change the link with an icon like the icon to access the group card.

Commit:
Add Gallery by group

coudot commented on htdocs/gallery.php:
```diff
@@ -23,8 +23,14 @@
 
+    # Search for users in group
+    if (isset($_GET['dn'])) {
+        $gallery_user_filter = substr($ldap_user_filter, 0, -1).'(memberOf='.$_GET['dn'].'))';
```
What if we defined gallery_user_filter in configuration? Here you are wiping this configuration value.

coudot commented on htdocs/gallery.php:
```diff
@@ -23,8 +23,14 @@
-    $gallery_filter = $ldap_user_filter;
     if (isset($gallery_user_filter) ) {
         $gallery_filter = $gallery_user_filter;
     }
```
I think you should adapt the filter here.

coudot commented on templates/listing_table.tpl:
```diff
@@ -33,6 +34,7 @@
         {/if}
         </td>
     {/foreach}
+        <td><a href="index.php?page=gallery&dn={$entry.dn|escape:'url'}">{$msg_gallery}</a></td>
```
Maybe we should use a GET parameter named "groupdn" to clearly says that we are displaying a gallery for a group.

coudot commented on templates/listing_table.tpl:
```diff
@@ -3,6 +3,7 @@
     <tr>
         <th class="hidden-print nosort">&nbsp;</th>
         {foreach $listing_columns as $item}<th>{$msg_label_{$item}}</th>{/foreach}
+        <th>{$msg_gallery}</th>
```
We will see how to improve this

Commit:
Changes :
  - GET param : dn to groupdn
  - gallery links to button
  - dont show gallery link in directory for user list
  - gallery_user_filter available
